CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+## Build and test
```

Comment:
@peterzhu2118 if you think this section looks good, I can add it to other repos as well.